tasks/internal_reconcile_report_orphan/test/unit_tests/adapters/storage/test_rdbms.py

```diff
@@ -126,11 +126,11 @@ def test_get_orphans_page_no_rows_happy_path(
         mock_make_url.assert_called_once_with(connection_uri)
         mock_create_engine.assert_called_once_with(mock_make_url.return_value, future=True)
 
-        logger = Mock()
+        LOGGER = Mock()
```

Comment:
Here and elsewhere, switch to lower-case. Upper case seems to denote class properties.